src/modules/components/transaction/transactionDataListItem/transactionDataListItemStructure/transactionDataListItemStructure.tsx

```diff
@@ -69,42 +69,34 @@ export const TransactionDataListItemStructure: React.FC<ITransactionDataListItem
         type === TransactionType.ACTION || tokenAmount == null ? '-' : `${formattedTokenValue} ${tokenSymbol}`;
 
     return (
-        <DataList.Item className={classNames('px-4 py-0 md:px-6', className)} href={parsedHref} {...otherProps}>
```

Comment:
Looks like the token price gets truncated
<img width="972" alt="Screenshot 2024-11-18 at 10 13 35" src="https://github.com/user-attachments/assets/bb8a4211-9f2c-4fcd-8912-74f03ef00d54">